tools/otsdb-dump-restore/README.md

```diff
@@ -1,26 +1,24 @@
-# OpenKilda TSDB dump/restore tools
+# OpenKilda TSDB dump/restore tool
 
 This packet contains python package that provide `kilda-otsdb-dump-restore` CLI tool. The tool itself
 provides following commands:
 * `kilda-otsdb-dump` download data from an OpenTSDB endpoint.
 * `kilda-otsdb-restore` restore data to an OpenTSDB capable database.
 
-# Create docker image
-
-```bash
-make
-````
+## Build
+`make prod` - builds a container image with the application, it labels it with kilda-otsdb-dump-restore
+`make devel` - builds a container and starts an interactive shell inside it (app directory is attached inside the container as a volume,
+so all changes made to the application code are available inside the container without the need to restart/rebuild it). Useful for the application debug.
```

Comment:
I'd recommend not to use so long sentences in parentheses. Those are 4 separate thoughts/sentences:
-  builds a container ... 
- App directory is attached ...
- All changes made ...
- Useful for debugging ...

Also, `debug` is a verb. It is better to say: It is useful for ... debugging.